src/Helpers/helpers.php

```diff
@@ -20,3 +20,10 @@ function voyager_asset($path, $secure = null)
         return asset(config('voyager.assets_path').'/'.$path, $secure);
     }
 }
+
+if (!function_exists('isRtl')) {
+    function isRtl()
+    {
+        return app()->getLocale() == 'fa' || app()->getLocale() == 'ar';
```

Comment:
I'm not sure I like having the language codes hard-coded here.  There are more languages than just Farsi and Arabic that are rtl. (https://meta.wikimedia.org/wiki/Template:List_of_language_names_ordered_by_code)

Can we move this to a config setting or something? I'm open to suggestions on this.

Reply:
you're right , i'll do it

Reply:
Perfect, thanks!